src/site/markdown/PackagingTypes.md

```diff
@@ -5,7 +5,7 @@ Tycho defines the following custom Maven packaging types targeted for Eclipse Pl
 * `eclipse-plugin` corresponds to [Eclipse Plug-in and Plug-in Fragment projects](https://wiki.eclipse.org/PDE/User_Guide#Plug-in).
 * `eclipse-test-plugin` is similar to `eclipse-plugin` but only contains Plugin Tests to be executed inside an OSGi runtime. There is a noticable difference between `eclipse-plugin` and `eclipse-test-plugin` with respect to the goal executed during `integration-test`. The former uses `tycho-surefire-plugin:integration-test` while the latter uses `tycho-surefire-plugin:test`.
 * `eclipse-feature` corresponds to [Eclipse Feature projects](https://wiki.eclipse.org/PDE/User_Guide#Feature)
-* `eclipse-repository` corresponds to [Eclipse Update Site projects](https://wiki.eclipse.org/PDE/User_Guide#Update_Site)
+* `eclipse-repository` corresponds to [Eclipse Update Site projects](https://wiki.eclipse.org/PDE/User_Guide#Update_Site) or project containing a `category.xml` file
```

Comment:
Actually `site.xml` projects are no longer supported since Tycho 3 (or 4? not sure).

Reply:
How about: * `eclipse-repository` corresponds to projects containing a `category.xml` file, the support of [Eclipse Update Site projects](https://wiki.eclipse.org/PDE/User_Guide#Update_Site) has been removed in latest Tycho versions